integrations/dashly/test/index.test.js

```diff
@@ -0,0 +1,426 @@
+'use strict';
+
+var Analytics = require('@segment/analytics.js-core').constructor;
+var clone = require('@ndhoule/clone');
+var integration = require('@segment/analytics.js-integration');
+var sandbox = require('@segment/clear-env');
+var tester = require('@segment/analytics.js-integration-tester');
+var Dashly = require('../lib/');
+
+describe('Dashly', function() {
+  var analytics;
+  var dashly;
+  var options = {
```

Comment:
As I understood from reading segment test article, they expect tests to work with real life dashly account. Will fake key work?